guides/doc-Administering_Red_Hat_Satellite/topics/proc_estimating-the-size-of-a-backup.adoc

```diff
@@ -44,7 +44,7 @@ The following table describes the compression ratio of all data items included i
 |Data type |Directory |Ratio |Example results
 
 |PostgreSQL database files
-|`/var/lib/pgsql/data`
+|`/var/opt/rh/rh-postgresql12/lib/pgsql/data`
```

Comment:
Given how often it's repeated, perhaps we should define the PG path in a variable? It's also slightly different on Debian (`/var/lib/postgresql/11` on Debian 10).